native/vincenty.rs

```diff
@@ -0,0 +1,134 @@
+#![allow(non_snake_case)]
+
+const RADIUS_AT_EQUATOR: f64 = 6378137.0;
+const FLATTENING_ELIPSOID: f64 = 1.0 / 298.257223563;
+const RADIUS_AT_POLES: f64 = (1.0 - FLATTENING_ELIPSOID) * RADIUS_AT_EQUATOR;
+const MAX_ITERATIONS: u32 = 200;
+const CONVERGENCE_THRESHOLD: f64 = 0.000000000001;
+const PRECISION: i32 = 6;
+
+pub type Coordinate = (f64, f64);
+
+pub fn distance(c1: Coordinate, c2: Coordinate) -> Option<f64> {
```

Comment:
Get rid of this?

Reply:
I think its ok, we use it in tests and lib.rs

Reply:
I meant as in it has the same exact signature as `vincenty_inverse`, so might as well make it one thing